pennylane/operation.py

```diff
@@ -1623,43 +1618,12 @@ def parameter_frequencies(self):
             "and parameter frequencies can not be computed as no generator is defined."
         )
 
-    @property
-    def inverse(self):
-        """Boolean determining if the inverse of the operation was requested."""
-        return False
-
-    def matrix(self, wire_order=None):
```

Comment:
Why do you remove the matrix method completely? Is not expected to stay in the abc class?

Reply:
We have an `Operator.matrix` method that does the same thing